devops/can/can_node.py

```diff
@@ -28,7 +28,7 @@
     #Create the thread for CAN
     can = DrvCanNodeC(tx_buffer_size= 150, working_flag=_working_can)
     try:
-        can.start()
+        can.run()
         while 1:
```

Comment:
Remove while, the node enters on an infinite while on run function. You never will reach while 1 statement.

Reply:
Join is not necessary. It is used only when a new thread is spawn.